### PR TITLE
setting up homework 1

### PR DESCRIPTION
**Pull request by theKyleBogucki**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+     background-color: navy;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+.container {
+  width: 960px;
```
Comment (lukeruth) [inline css/style.css:63]: Now that we've gone through responsive - in your next draft this should become max-width so it adjusts with the page.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+
+
+header {
+  background: #0a1629;
+  padding: 5px;
+}
+
+
+.navigation {
+  height: 30px;
```
Comment (lukeruth) [inline css/style.css:75]: Is there a reason to give this a fixed height? Why not let the content dictate that instead?

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+  font-size: 14px;
+  font-weight: 700;
+  color: white;
+}
+
+
+.navigation ul {
+  padding: 4px 5px 0px 0px;
```
Comment (lukeruth) [inline css/style.css:83]: Remember, there's an odd quirk in CSS where values of 0 are actually unitless - so it should be `padding: 4px 5px 0 0`

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+}
+
+
+.navigation li {
+  display: inline;
```
Comment (lukeruth) [inline css/style.css:88]: This might be better suited as `inline-block` so that padding and margin behave correctly on the top and bottom.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,439 @@
...
+  padding: 8px;
+}
+
+
+/******************************************
+/* home page
+/*******************************************/
+
+.main-search {
+  padding-top: 200px;
```
Comment (lukeruth) [inline css/style.css:104]: This seems like you might be using padding to accomplish something padding wasn't intended to do. If you're centering it vertically try flexbox.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,439 @@
...
+}
+
+
+.titleTxt {
+  text-align: center;
+  font-weight: 700;
+  color: white;
+  font-size: 72px;
```
Comment (lukeruth) [inline css/style.css:112]: Now that we've learned about responsive font units, try converting these pixel values to ems or rems in the next draft.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,439 @@
...
+  color: white;
+  text-decoration: none;
+  font-size: 20px;
+  margin-bottom: 15px;
+  display: inline-block;
+}
+
+
+.info a:hover {
+  text-decoration: underline;
```
Comment (lukeruth) [inline css/style.css:183]: Nice use of the hover psuedo selector

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,439 @@
...
+  padding: 10px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+
+.iceTime {
+  border-bottom: 1px solid #0a1629;
+  padding: 10px;
+  background: rgba(255,255,255,.9);
```
Comment (lukeruth) [inline css/style.css:295]: Good use of RGBa throughout to create a subtle transparency effect.

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,439 @@
...
+    table-layout: fixed;
+    border-collapse: collapse;
+    width: 100%;
+}
+
+
+th {
+    padding: 0 1px;
+    text-align: center;
+    background-color:gray;
```
Comment (lukeruth) [inline css/style.css:342]: Try using hex codes instead of color keywords.

File: grid.html [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+<html lang="en">
+	<head>
+
+    	<meta charset="utf-8">
+    	<meta name="description" content="Ice Times">
+    	<meta name="keywords" content="Hockey, Ice Hockey, Pick-Up, Stick 'n' Shoot">
+
+		<title>Ice Times</title>
+
+		<link rel="icon" type="image/ico" href="images/hockey.png"/>
```
Comment (lukeruth) [inline grid.html:11]: Nice! Definitely a professional touch.

File: grid.html [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+
+    <header>
+      <div class="container">
+
+        <div class="navigation">
```
Comment (lukeruth) [inline grid.html:24]: Do some research on the `<nav>` tag - may be more appropriate here.

File: grid.html [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+            <h1>Rockville Ice Arena</h1>
+            <p>SnS 1:15pm - 2:30pm</p>
+            <p>SnS 4:00pm - 5:10pm</p>
+        </a>
+      </div>
+      <div class="row clearfix">
+        <a  class="tile" href="rinks/reston.html">
+            <h1>Skatequest</h1>
+            <p>PU 9:00am - 10:20am</p>
+            <p>&nbsp</p>
```
Comment (lukeruth) [inline grid.html:80]: When you're encoding values like this they need to end in a semicolon - `&nbsp;`

File: index.html [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+      <div class="container">
+
+        <div class="titleTxt">
+          Welcome to Ice Times
+        </div>
+
+        <div class="search">
+          <!-- <h1> Search by rink </h1> -->
+          <input type="titleTxt" id="rinkSearch"  placeholder="Type rink name here"></input>
+          <div class="button" id="rinkSubmit">
```
Comment (lukeruth) [inline index.html:46]: If this button is behaving as a link (taking you somewhere else), semantically it's probably better served as an `<a>` tag.

File: map.html [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact Us</a></li>
+          </ul>
+        </div>
+
+      </div>
+    </header>
+  <body>
+    <div class="background_map">
+      <iframe src="https://www.google.com/maps/d/embed?mid=zWozML4ch9uM.kE6ZQEzTxD7I" width="100%" height="1000px"></iframe>
```
Comment (lukeruth) [inline map.html:52]: Nice use of an embedded map!

File: resources/mapsed-master/examples/06-full-example.js [hunk 15]
```diff
@@ -0,0 +1,324 @@
...
+	// wire up examples
+	
+	// wire up full window example
+	$("#show-full-window")
+		.on("click", fullWindowExample)
+		//.trigger("click")
+	;
+	
+});
+
```
Comment (lukeruth) [inline resources/mapsed-master/examples/06-full-example.js:324]: Are all these examples from some plugin you pulled in? If so, generally they should be kept outside of the "production" environment of your site. Rule of thumb is to only sync functional code.

File: rinks/cabinjohn.html [hunk 16]
```diff
@@ -0,0 +1,388 @@
...
+
+					<div class="actions">
+
+						<a href="#"> Favorite this rink </a>
+						<a href="#"> Print full schedule </a>
+						<a href="#"> Share this page </a>
+
+					</div>
+
+					<div class="nearByRinks">
```
Comment (lukeruth) [inline rinks/cabinjohn.html:83]: Nice clear class names make it obvious what section of the page you're styling.

File: rinks/cabinjohn.html [hunk 17]
```diff
@@ -0,0 +1,388 @@
...
+								<div class="iceTime">
+									<h1>2:00pm - 3:00pm</h1>
+									<h2>10 skaters + 1 goalie(s) registered</h2>
+									<a href="#">Sign-up</a>
+								</div>
+
+								<div class="iceTime">
+									<h1>3:15pm - 4:45pm</h1>
+									<h2>16 skaters + 2 goalie(s) registered</h2>
+									<a href="#">Sign-up</a>
```
Comment (lukeruth) [inline rinks/cabinjohn.html:150]: As you already know, the real challenge will be in dynamically grabbing and populating this data, instead of hard-coding it.

File: rinks/reston.html [hunk 18]
```diff
@@ -0,0 +1,368 @@
+<!DOCTYPE html>
```
Comment (lukeruth) [inline rinks/reston.html:1]: These rink pages all look good. Nice use of a consistent layout (template) for each one.

File: rinks/wheaton.html [hunk 19]
```diff
@@ -0,0 +1,368 @@
...
+			</div> <!--end right column-->
+
+		</div> <!--end container-->
+
+
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="js/index.js"></script>
+  </body>
+  </html>
```
Comment (lukeruth) [inline rinks/wheaton.html:368]: Really great job overall. I like the organization of the different pages, the templating, the styles. It's all really coming together. Keep up the good work and let me know if you have any questions. 

Merge this whenever you're ready!
